### PR TITLE
[Feature] Add `FontConnectorController`

### PR DESCRIPTION
**Pull request by Matthiee**
This PR adds `FontConnectorController` and related types. 

## Related tickets

-   [EDT-563](https://support.chili-publish.com/browse/EDT-563)

### CODE REVIEW

**Review by pietervp — APPROVED** [review]
There are some linting warnings you will need to fix before merging, functionally LGTM!

**Review by pietervp — APPROVED** [review]

**Review by brapoprod — APPROVED** [review]

**Pull request merged**
